_posts/2021-08-11-Day-3-Post.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+title: "Monte Carlo again? It's a Loop"
+author: Kevin Fong
+---
+## Continuing on Monte Carlo Simulation
+
+Our primary topic today focuses on the application of the Monte Carlo simulation to generate sample configurations of particles in order understand and charcterize the statistical mechanics of a system. However as powerful as Monte Carlo simulation is, simply considering all coordinates for our configuration is very inefficient. This is because although a randomly generated configuration may be possible, configurations with high energy are simply not likely to occur. Therefore is useful to sample positions from a desired equilibrium probability distribution, where configurations that have low probabilty are generated less frequently to more realistic and revelent configurations. This is the concept of **importance sampling**.
```

Comment:
```suggestion
Our primary topic today focuses on the application of the Monte Carlo simulation to generate sample configurations of particles in order understand and characterize the statistical mechanics of a system. However as powerful as Monte Carlo simulation is, simply considering all coordinates for our configuration is very inefficient. This is because although a randomly generated configuration may be possible, configurations with high energy are simply not likely to occur. Therefore is useful to sample positions from a desired equilibrium probability distribution, where configurations that have low probability are generated less frequently to more realistic and relevent configurations. This is the concept of **importance sampling**.
```